### PR TITLE
CPDP-2206: add penalty warnings

### PR DESCRIPTION
**Pull request by mikail-khan**

### CODE REVIEW

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/confirmation_of_submission.scala.html [hunk 1]
```diff
@@ -50,6 +50,29 @@
 @isAgent = @{ request.userType.contains(UserType.Agent) }
 @isATrust = @{ justSubmittedReturn.subscribedDetails.isATrust }
 
+@completionDate =  @{justSubmittedReturn.completeReturn.fold(
+                        _.triageAnswers.fold(
```
Comment (AndyHWChung) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/confirmation_of_submission.scala.html:54]: you don't need to fold here and below - `_.triageAnswers" shold be complete so you can get a non-optional completion date from all the cases

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/confirmation_of_submission.scala.html [hunk 2]
```diff
@@ -103,6 +126,14 @@
   case _ => s"$key$userKey.printPage"
   }}
 
+@isLateReturn = @{
+    if(completionDate._1.map(d => d.plusDays(30L).isAfter(TimeUtils.today)).getOrElse(false)){
```
Comment (AndyHWChung) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/confirmation_of_submission.scala.html:130]:
```scala
if(x) true else false
``` 
is equivalent to just 
```scala
x
```

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/confirmation_of_submission.scala.html (hunk 1 above)
Comment (mikail-khan) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/confirmation_of_submission.scala.html]: ripped out the fold and did it via case analysis - much easier to read.

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/confirmation_of_submission.scala.html (hunk 2 above)
Comment (mikail-khan) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/confirmation_of_submission.scala.html]: yes, fixed.

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/confirmation_of_submission.scala.html [hunk 3]
```diff
@@ -216,6 +232,20 @@ <h2 class="heading-medium">@messages(s"$key$userKey.whatNext")</h2>
...
+                @if(isLateReturn){
+                  <p>@messages(s"$key.agent.penalty.warning"))</p>
+                }
+              }else if (isATrust) {
+                @if(isLateReturn){
+                    <p>@messages(s"$key.trust.penalty.warning"))</p>
+                }
+              }else{
+                @if(isLateReturn){
+                    <p>@messages(s"$key$userKey.penalty.warning"))</p>
```
Comment (mikail-khan) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/confirmation_of_submission.scala.html:245]: @nubz can you check if any css class needs to be applied to the penalty warning messages?

**Comment by AndyHWChung** [comment]
LGTM

**Pull request merged**